domain/src/main/kotlin/no/nav/su/se/bakover/domain/behandling/Behandlingsinformasjon.kt

```diff
@@ -0,0 +1,337 @@
+            val begrunnelse: String?
+        )
+    }
+
+    data class OppholdIUtlandet(
+        val status: Status,
+        val begrunnelse: String?
+    ) : Base(), DtoConvertable<OppholdIUtlandet.Dto> {
+        enum class Status {
+            SkalVæreMerEnn90DagerIUtlandet,
+            SkalHoldeSegINorge
+        }
+
+        override fun isValid(): Boolean =
+            when (status) {
+                Status.SkalVæreMerEnn90DagerIUtlandet -> begrunnelse != null
+                Status.SkalHoldeSegINorge -> begrunnelse == null
+            }
+
+        override fun isComplete(): Boolean = true
+
+        override fun toDto(): Dto = Dto(status, begrunnelse)
+
+        class Dto(
+            val status: Status,
+            val begrunnelse: String?
+        )
+    }
+
+    data class Formue(
+        val status: Status,
+        val verdiIkkePrimærbolig: Int?,
+        val verdiKjøretøy: Int?,
+        val innskudd: Int?,
+        val verdipapir: Int?,
+        val pengerSkyldt: Int?,
+        val kontanter: Int?,
+        val depositumskonto: Int?
+    ) : Base(), DtoConvertable<Formue.Dto> {
+        enum class Status {
+            Ok,
+            MåInnhenteMerInformasjon
+        }
+
+        override fun isValid(): Boolean =
+            when (status) {
+                Status.Ok ->
+                    verdiIkkePrimærbolig != null &&
+                        verdiKjøretøy !== null &&
+                        innskudd !== null &&
+                        verdipapir !== null &&
+                        pengerSkyldt !== null &&
+                        kontanter !== null &&
+                        depositumskonto !== null
+                Status.MåInnhenteMerInformasjon -> true
+            }
+
+        override fun isComplete(): Boolean = status != Status.MåInnhenteMerInformasjon
+
+        override fun toDto(): Dto = Dto(
+            status,
+            verdiIkkePrimærbolig,
+            verdiKjøretøy,
+            innskudd,
+            verdipapir,
+            pengerSkyldt,
+            kontanter,
+            depositumskonto
+        )
+
+        data class Dto(
+            val status: Status,
+            val verdiIkkePrimærbolig: Int?,
+            val verdiKjøretøy: Int?,
+            val innskudd: Int?,
+            val verdipapir: Int?,
+            val pengerSkyldt: Int?,
+            val kontanter: Int?,
+            val depositumskonto: Int?
+        )
+    }
+
+    data class PersonligOppmøte(
+        val status: Status,
+        val begrunnelse: String?
+    ) : Base(), DtoConvertable<PersonligOppmøte.Dto> {
+        enum class Status {
+            MøttPersonlig,
+            Verge,
+            FullmektigMedLegeattest,
+            FullmektigUtenLegeattest,
+            IkkeMøttOpp
+        }
+
+        override fun isValid(): Boolean = true
+        override fun isComplete(): Boolean = true
+
+        override fun toDto(): Dto = Dto(status, begrunnelse)
+
+        data class Dto(
```

Comment:
Trenger kanskje ikke Dto-er for data classes med val´s

Reply:
Vil vi ikke da også få med feltene `complete` og `valid`?
Er veldig for å kutte ut alt DTO-opplegget hvis mulig!